test/tests/ver-1dd/comparison_ver-1dd.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+import os
+import git
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+num_summation_terms = 1000
+N_o = 3.1622e22             # For convenience lattice density chosen as 3.1622e22 atom/m^3
+c_o = 0.0001                # dissolved gas atom fraction (-)
+D = 1                       # diffusivity (m^2/s)
+l = 1                       # slab thickness (m)
+
+# Extract data from 'gold' TMAP8 run
+tmap_sol = pd.read_csv(os.path.join(git.Repo('.',search_parent_directories=True).working_tree_dir, "test/tests/ver-1dd/gold/ver-1dd_out.csv"))
+tmap_time = np.array(tmap_sol['time'])
+tmap_prediction = np.array(tmap_sol['scaled_outflux'])
+idx = np.where(tmap_time >= 0.01)[0][0]
+
+# Calculate the breakthrough time from numberical solution
+tmap_slope = (tmap_prediction[idx+1:] - tmap_prediction[idx:-1]) / (tmap_time[idx+1:] - tmap_time[idx:-1])
+tmap_intercept = tmap_time[np.argmax(tmap_slope)+idx] -  (tmap_prediction[int(np.argmax(tmap_slope)+idx)]) / np.max(tmap_slope)
+output_line = f"The breakthrough time from numerical solution is {tmap_intercept:.2f} s"
+print(output_line)
+
+# analytical solution
+analytical_time = np.array(tmap_time)
+tau_be = l**2 / (2 * (np.pi)**2 * D)  # calculate analytical breakthrough time
+output_line = f"The breakthrough time from analytical solution is {tau_be:.2f} s"
+print(output_line)
+
+def summation_term(num_terms, time):
+    sum = 0.0
+    for m in range(1, num_terms):
+        sum += (-1)**m * np.exp(-1 * m**2 * time / (2*tau_be))
+    return sum
+# Calculate the analytical solution
+analytical_flux = N_o * (c_o * D / l) * \
+                  (1 + 2 * summation_term(num_summation_terms, analytical_time))
+
+# Plot figure for verification
+fig = plt.figure(figsize=[6.5, 5.5])
+gs = gridspec.GridSpec(1, 1)
+ax = fig.add_subplot(gs[0])
+
+ax.plot(analytical_time, analytical_flux, label=r"Analytical", c='k', linestyle='--') # numerical solution
+ax.plot(tmap_time, tmap_prediction, label=r"TMP8", c='tab:gray') # numerical solution
+ax.plot([tau_be, tau_be], [0, 3.15e18], label=r"Analytical breakthrough time", c='tab:brown', linestyle='--')
+ax.plot([tmap_intercept, 0.221], [0, (0.221 - tmap_intercept) * np.max(tmap_slope)],
+        label=r"Numerical breakthrough time", c='tab:brown')
+ax.set_xlabel(u'Time(s)')
+ax.set_ylabel(u"Flux (atom/m$^2$s)")
+ax.legend(loc="best")
+ax.set_xlim(left=0)
```

Comment:
```suggestion
ax.set_xlim(left=0, right=1.5)
```
Otherwise everything is on the left of the graph.